rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -941,7 +926,7 @@ private static Federation createNonStandardErpFederation() {
             federationArgs,
             erpPubKeys,
             activationDelay,
-            activations
+            ActivationConfigsForTest.all().forBlock(0L)
```

Comment:
use class level activations?

Reply:
Class level activations gets modified in a couple of places, creates some issues here

Reply:
oki doki